core/src/Cache/Keys/MsalAccountCacheKey.cs

```diff
@@ -76,17 +76,17 @@ public string GetiOSAccountKey()
 
             stringBuilder.Append(_environment);
 
-            return stringBuilder.ToString();
+            return stringBuilder.ToString().ToLower();
         }
 
         public string GetiOSServiceKey()
         {
-            return _tenantId ?? "";
+            return (_tenantId ?? "").ToLower();
```

Comment:
>     return (_tenantId ?? "").ToLower(); [](start = 7, length = 40)

I'm speculating, however wouldn't this righ backwards compatibility?

Reply:
we do not access cache by individual key , we do it by cache entity type - like read all access tokens,
so this is not a problem